credentials_template.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I see we have a config file but at the same time, the method needs to have the __init__ of the connector takes the input as constructor argument. This is not consistent

Reply:
I had created the credentials dictionary initially, but later changed to taking input as arguments to the constructor. So will delete this file.